yt_dlp/extractor/weyyak.py

```diff
@@ -0,0 +1,50 @@
+import re
```

Comment:
Why's this file in this PR?

Reply:
This was the other PR, it somehow linked to this..